torch/onnx/symbolic_helper.py

```diff
@@ -437,6 +437,27 @@ def assert_training_mode(op_mode, op_name):
                       op_mode + " mode. The model will be exported in " +
                       training_mode + ", as specified by the export mode.")
 
+def _flatten_helper(g, input, start_dim, end_dim, dim):
+    # use Reshape for cases where the output shape is not 2D
+    if not input.isCompleteTensor():
```

Comment:
no longer needed, since we are dynamically operating on shape produced by "Shape" node.